gst/nnstreamer/tensor_common.c

```diff
@@ -794,11 +814,18 @@ gst_tensors_config_validate (const GstTensorsConfig * config)
 
   /* framerate (numerator >= 0 and denominator > 0) */
   if (config->rate_n < 0 || config->rate_d <= 0) {
+    nns_logw ("Failed to validate tensors config. framerate: %d/%d. "
+        "framerate should be numerator >= 0 and denominator > 0.",
+        config->rate_n, config->rate_d);
```

Comment:
This should be debug, not warning. You don't know the caller's context.

The caller wants to know if it's valid or not. The caller might intentionally test invalid tensors (to do something else if it's invalid).